components/script/dom/htmlmediaelement.rs

```diff
@@ -1280,6 +1288,21 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
         self.media_element_load_algorithm();
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-media-muted
+    fn GetMuted(&self) -> bool {
+        self.muted.get() as bool
+    }
+
+    // https://html.spec.whatwg.org/multipage/#dom-media-muted
+    fn SetMuted(&self, value: bool) {
+        self.muted.set(value);
+        let window = window_from_node(&*elem);
+        window
+            .task_manager()
+            .media_element_task_source()
+            .queue_simple_event(self.upcast(), atom!("volumechange"), &window);
```

Comment:
This is likely failing to build because we need to list the new atom in [components/atoms/static_atoms.txt](https://github.com/servo/servo/blob/master/components/atoms/static_atoms.txt)